app/cafe.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+
+
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("All friends should be vaccinated")
```

Comment:
provide more descriptive error messages. Not all friends should be vaccinated, but exact one

Reply:
Fixed.. I guess)